test/e2e/flask/solana-standard/connect.spec.ts

```diff
@@ -0,0 +1,355 @@
+import { strict as assert } from 'assert';
```

Comment:
Can we rename the folder to `solana-wallet-standard` rather than `solana-standard`?